core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaFactory.java

```diff
@@ -83,8 +79,8 @@ public Producer<byte[], byte[]> takeProducer() {
     }
 
     /**
-     * Release kafka producer that was obtained by {@link #takeProducer()} method. If producer was not obtained by
-     * {@link #takeProducer()} call - method will throw {@link NullPointerException}
+     * Release kafka producer that was obtained by {@link #takeProducer(String)} method. If producer was not obtained by
+     * {@link #takeProducer(String)} call - method will throw {@link NullPointerException}
```

Comment:
Never noticed, but the comment about NPE is wrong (in general, and as in not being up to date with the actual code).